src/wallet/rpcwallet.cpp

```diff
@@ -2798,9 +2795,15 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
             coinControl.signalRbf = options["replaceable"].get_bool();
         }
         if (options.exists("conf_target")) {
-            coinControl.nConfirmTarget = options["conf_target"].get_int();
+            if (coinControl.m_feerate) {
```

Comment:
Just because if you change order (handle `feeRate` option below) it still works.

Nit, s/fundrawtx/fundrawtransaction.